app/src/org/commcare/views/widgets/DatePrototypeFactory.java

```diff
@@ -0,0 +1,19 @@
+package org.commcare.views.widgets;
+
+import android.content.Context;
+
+import org.javarosa.form.api.FormEntryPrompt;
+
+public class DatePrototypeFactory {
```

Comment:
I assume that this factory was useful when testing multiple prototypes. Now that we have settled on one, does it make sense to remove it and explicitly construct the `GregorianDateWidget` in `WidgetFactory`?